src/main/java/com/afornalik/ox/view/draw/UIDrawBoardSideNumerate.java

```diff
@@ -0,0 +1,92 @@
+package com.afornalik.ox.view.draw;
+
+import com.afornalik.ox.model.board.Board;
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.board.OutOfBoardException;
+
+import java.math.BigInteger;
+
+/**
+ * Class responsible for print board on console.
+ * board have two side numerated axis.
+ *
+ * @author Andrzej Fornalik
+ */
+
+public class UIDrawBoardSideNumerate implements UIDrawBoard {
+
+    private Board board;
+    private BigInteger boardDimension;
+
+    /**
+     * Constructor with parameter board.
+     * This constructor also calculate boardDimension variable.
+     *
+     * @param board Board This parameter represent board in OX game
+     */
+
+    public UIDrawBoardSideNumerate(Board board) {
+        this.board = board;
+        this.boardDimension = new BigInteger(String.valueOf(board.getBorderSize()))
```

Comment:
Again, are you need BigInteger for this?

Reply:
Replaced with int type.